app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -327,7 +323,7 @@ protected String getTrackedViewName() {
         } catch (IndexOutOfBoundsException e) {
             pageName = "";
         }
-        Chapter chapter = mChapterAdapter.findById(selectedChapterId);
+        Chapter chapter = findChapterById(selectedChapterId);
```

Comment:
I think as a field we should also store `selectedChapter` as a whole object instead of just the id. I wanted to do it but I think it will blow up this PR.